runtime/doc/news.txt

```diff
@@ -23,7 +23,12 @@ The following changes may require adaptations in user config or plugins.
 • "#" followed by a digit no longer stands for a function key at the start of
   the lhs of a mapping.
 
-• `:behave` was removed. if you used `:behave mswin`, the following is equivalent: >vim
+• `:behave` was removed.
+  - If you used `:behave xterm`, the following is equivalent: >vim
+
+    set mousemodel=extend
+<
+  - If you used `:behave mswin`, the following is equivalent: >vim
```

Comment:
I think I might not have spaced this right; the current Neovim website doesn't have the indentation for the `behave mswin` item here:

![image](https://github.com/neovim/neovim/assets/658538/df25e83d-864d-42f5-a3a8-ae275fbe58dc)

I wanted this to be at the same level as `behave xterm`, but apparently it's been outdented. I'm guessing I need another newline between the end of the code block and the new bullet point?

Reply:
I think line 30 would need to look like:

    < - If you used `:behave mswin`, the following is equivalent: >vim